cloudbridge/cloud/providers/gce/provider.py

```diff
@@ -73,18 +74,10 @@ def gce_compute(self):
 
     def _connect_gce_compute(self):
         if self.credentials_file:
-            with open(self.credentials_file) as f:
-                data = json.load(f)
-                credentials = SignedJwtAssertionCredentials(
-                    data['client_email'], data['private_key'],
-                    'https://www.googleapis.com/auth/compute')
+            credentials = GoogleCredentials.from_json(self.credentials_file)
```

Comment:
Done. I now use ServiceAccountCredentials to handle json file for the service account key. This is intended to be used only for cases where GoogleCredentials (the preferred solution) is not applicable.

The service account key json file I used for testing was downloaded from:
console.cloud.google.com --> APIs --> Credentials --> Create credentials | Service account key --> Compute Engine default service account